packages/superset-ui-chart/src/controls/NVD3.ts

```diff
@@ -0,0 +1,299 @@
+/**
```

Comment:
I think this could be a part of @superset-ui/legacy-preset-chart-nvd3 since it's written for (and only used by) that one plugin. 

Same applies to the DeckGL equivalent - those shared controls could live with that plugin.

Reply:
Oh thanks for the pointer, if it's only used there then that's way better.

Reply:
Agree with @rusackas